src/Symfony/Bundle/WebServerBundle/Command/ServerLogCommand.php

```diff
@@ -47,11 +47,24 @@ protected function configure()
         }
 
         $this
-            ->setDescription('Start a log server that displays logs in real time')
             ->addOption('host', null, InputOption::VALUE_REQUIRED, 'The server host', '0:9911')
             ->addOption('format', null, InputOption::VALUE_REQUIRED, 'The line format', ConsoleFormatter::SIMPLE_FORMAT)
             ->addOption('date-format', null, InputOption::VALUE_REQUIRED, 'The date format', ConsoleFormatter::SIMPLE_DATE)
             ->addOption('filter', null, InputOption::VALUE_REQUIRED, 'An expression to filter log. Example: "level > 200 or channel in [\'app\', \'doctrine\']"')
+            ->setDescription('Starts a log server that displays logs in real time')
+            ->setHelp(<<<'EOF'
+<info>%command.name%</info> starts a log server to display in real time the log
+messages generated by your application:
+
+  <info>php %command.full_name%</info>
+
+To get the information as a machine readable format, use the
+<comment>--filter</> option:
+
+<info>php %command.full_name% --filter=port</info>
+
+Supported values are <comment>port</>, <comment>host</>, and <comment>address</>.
```

Comment:
that's wrong actually. `filter` is about filtering logs with the ExpressionLanguage